<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StreamsPage</class>
 <widget class="QWidget" name="StreamsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>259</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="vlayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="viewStack">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="ItemView" name="view"/>
     <widget class="ItemView" name="searchView"/>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="diStatusLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>16</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="SizeWidget" name="sizeWidget"/>
     </item>
     <item>
      <widget class="MenuButton" name="menuButton"/>
     </item>
     <item>
      <widget class="ToolButton" name="replacePlayQueue"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ItemView</class>
   <extends>QTreeView</extends>
   <header>itemview.h</header>
  </customwidget>
  <customwidget>
   <class>MenuButton</class>
   <extends>QToolButton</extends>
   <header>menubutton.h</header>
  </customwidget>
  <customwidget>
   <class>ToolButton</class>
   <extends>QToolButton</extends>
   <header>toolbutton.h</header>
  </customwidget>
  <customwidget>
   <class>SizeWidget</class>
   <extends>QWidget</extends>
   <header>sizewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
